generators/app/templates/client/server.js

```diff
@@ -0,0 +1,22 @@
+require('./config/env');
+var path = require('path');
+var url = require('url');
+var proxy = require('proxy-middleware');
+var history = require('connect-history-api-fallback');
+var serveStatic = require('express').static;
+var server = require('express')();
+
+var port = process.env.NODE_ENV == 'production' ? process.env.PORT : 8080;
+
+server.use(history());
+server.use('/', serveStatic(path.join(__dirname, 'public')));
+
+if(process.env.NODE_ENV !== 'production') {
+  server.use('/', proxy(url.parse(process.env.WEBPACK_DEV_SERVER_URL)))
+}
+
+server.use('/', proxy(url.parse(process.env.API_URL)));
```

Comment:
Not really.
I don't think it will add much load because it's just a simple proxy, and local, too. A heavy query running for 1 second will add much more load to all parties involved than that proxy.